package.json

```diff
@@ -72,6 +72,9 @@
       "svelte": "$svelte",
       "vite": "$vite",
       "@types/node@<=20.12.0": "20.11.5"
-    }
+    },
+    "onlyBuiltDependencies": [
+      "esbuild"
```

Comment:
pnpm@9 changed the default for enable-pre-post scripts to true. this changes it so that only scripts we trust get run. for now only esbuild.